scripts/global.js

```diff
@@ -0,0 +1,1200 @@
+        ),
+        domStakingRewardsTitle: document.getElementById(
+            'staking-rewards-title'
+        ),
+        domMnemonicModalContent: document.getElementById(
+            'ModalMnemonicContent'
+        ),
+        domMnemonicModalButton: document.getElementById(
+            'modalMnemonicConfirmButton'
+        ),
+        domExportDiv: document.getElementById('exportKeyDiv'),
+        domExportPublicKey: document.getElementById('exportPublicKeyText'),
+        domExportPrivateKeyHold: document.getElementById('exportPrivateKey'),
+        domExportPrivateKey: document.getElementById('exportPrivateKeyText'),
+        domExportWallet: document.getElementById('guiExportWallet'),
+        domWipeWallet: document.getElementById('guiWipeWallet'),
+        domRestoreWallet: document.getElementById('guiRestoreWallet'),
+        domNewAddress: document.getElementById('guiNewAddress'),
+        domConfirmModalHeader: document.getElementById('confirmModalHeader'),
+        domConfirmModalTitle: document.getElementById('confirmModalTitle'),
+        domConfirmModalContent: document.getElementById('confirmModalContent'),
+        domConfirmModalButtons: document.getElementById('confirmModalButtons'),
+        domConfirmModalConfirmButton: document.getElementById(
+            'confirmModalConfirmButton'
+        ),
+        domConfirmModalCancelButton: document.getElementById(
+            'confirmModalCancelButton'
+        ),
+
+        masternodeLegacyAccessText:
+            'Access the masternode linked to this address<br> Note: the masternode MUST have been already created (however it can be online or offline)<br>  If you want to create a new masternode access with a HD wallet',
+        masternodeHDAccessText:
+            "Access your masternodes if you have any! If you don't you can create one",
+        // Aggregate menu screens and links for faster switching
+        arrDomScreens: document.getElementsByClassName('tabcontent'),
+        arrDomScreenLinks: document.getElementsByClassName('tablinks'),
+        // Alert DOM element
+        domAlertPos: document.getElementsByClassName('alertPositioning')[0],
+        domNetwork: document.getElementById('Network'),
+        domNetworkE: document.getElementById('NetworkE'),
+        domNetworkD: document.getElementById('NetworkD'),
+        domDebug: document.getElementById('Debug'),
+        domTestnet: document.getElementById('Testnet'),
+        domExplorerSelect: document.getElementById('explorer'),
+        domNodeSelect: document.getElementById('node'),
+        domTranslationSelect: document.getElementById('translation'),
+    };
+    loadImages();
+    let localTranslation = localStorage.getItem('translation');
+    // Check if set in local storage
+    if (localTranslation != null) {
+        switchTranslation(localTranslation);
+    } else {
+        // Check if we support the user's browser locale
+        if (arrActiveLangs.includes(strLang)) {
```

Comment:
arrActiveLangs and strLang are being considered undefined in visual studio but it is defined in index.template.html